src/google/schema.js

```diff
@@ -47,6 +47,7 @@ const schema = {
   resp_http_Content_Type: '%{resp.http.Content-Type}V',
   service_config: 'undefined',
   status_code: '%>s',
+  resp_http_x_trace: '%{req.http.x-trace}V',
```

Comment:
Not a good idea: we should use the BigQuery logs for traffic analysis and the trace isn't useful there, but will inflate the amount of data we store (and slow down searches and results in addition to costing us money).